src/Resources/views/admin/customers/partials/list_table.html.twig

```diff
@@ -5,7 +5,7 @@
             <th class="table-id-column">#</th>
             <th width="80px">
                 <div class="pos">
-                    <a class="sorting{{ cmf_currentOrder('o_id') }}" href="{{ path('customermanagementframework_admin_customers_list', cmf_nextFormOrderParams('o_id')) }}">
+                    <a class="sorting{{ cmf_currentOrder('id') }}" href="{{ path('customermanagementframework_admin_customers_list', cmf_nextFormOrderParams('id')) }}">
```

Comment:
@dvesh3 @mcop1 also on twig files same thing as https://github.com/pimcore/customer-data-framework/pull/379/files#r1039488239

Reply:
we should pass this variable (with value from `getVersionDependentDatabaseColumnName`) from list action and read it here..